jerry-core/include/jerryscript-core.h

```diff
@@ -522,6 +522,8 @@ jerry_length_t jerry_get_typedarray_length (jerry_value_t value);
 jerry_value_t jerry_get_typedarray_buffer (jerry_value_t value,
                                            jerry_length_t *byte_offset,
                                            jerry_length_t *byte_length);
+jerry_value_t jerry_json_parse (const char * json_string);
```

Comment:
I think it would be more consistent with the rest of the API if this would take a `jerry_value_t` representing the string.